loopchain/store/key_value_store_plyvel.py

```diff
@@ -13,16 +13,11 @@
 # limitations under the License.
 
 import functools
-import json
-import logging
 import urllib.parse
-from typing import Union
 
 import plyvel
 
-from loopchain import configure_default as conf
-from loopchain.blockchain.transactions import TransactionSerializer, TransactionVersioner
-from loopchain.blockchain.types import Hash32
+from loopchain.blockchain.transactions import TransactionVersioner
```

Comment:
It may be out of topic, but you can remove `KeyValueStorePlyvel.__tx_versioner`.
Then you can remove `TransactionVersioner` import line.

> not obligated. if you do not want to do this, I will handle it.
> Because it is caused by me. 😅

Reply:
👌